platform/viewer/cypress/integration/measurement-tracking/OHIFContextMenuCustomization.spec.js

```diff
@@ -0,0 +1,42 @@
+describe('OHIF Context Menu', function () {
+  beforeEach(function () {
+    cy.checkStudyRouteInViewer(
+      '1.2.840.113619.2.5.1762583153.215519.978957063.78'
+    );
+
+    cy.expectMinimumThumbnails(3);
+    cy.initCommonElementsAliases();
+    cy.initCornerstoneToolsAliases();
+    cy.resetViewport().wait(50);
+  });
+
+  it('checks context menu customization', function () {
+    // Add length measurement
+    cy.addLengthMeasurement();
+    cy.get('[data-cy="prompt-begin-tracking-yes"]').click();
+    cy.get('[data-cy="measurement-item"]').click();
+
+    const [x1, y1] = [150, 100];
+    cy.get('@viewport')
```

Comment:
I suspect we are doing a 'down' and 'up' here to avoid  the race condition (of which I apologize) if a simple click were done instead? Is there not a way to wait for a condition to be true in Cypress?

Reply:
That is right click, and I don't  know how to invoke click on a right click.